<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>config_dialog</class>
 <widget class="QDialog" name="config_dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>233</width>
    <height>330</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>99999</width>
    <height>99999</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Конфигурация измерения</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../irspy/qt/resources/icons.qrc">
    <normaloff>:/icons/icons/graph_2.png</normaloff>:/icons/icons/graph_2.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="load_from_file_button">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../irspy/qt/resources/icons.qrc">
         <normaloff>:/icons/icons/open.png</normaloff>:/icons/icons/open.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save_to_file_button">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../irspy/qt/resources/icons.qrc">
         <normaloff>:/icons/icons/save.png</normaloff>:/icons/icons/save.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="add_cmd_button">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../irspy/qt/resources/icons.qrc">
         <normaloff>:/icons/icons/plus.png</normaloff>:/icons/icons/plus.png</iconset>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="remove_cmd_button">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../irspy/qt/resources/icons.qrc">
         <normaloff>:/icons/icons/minus2.png</normaloff>:/icons/icons/minus2.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="cmd_edit">
     <property name="inputMask">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Введите команду</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="cmd_list_widget"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ok_button">
       <property name="text">
        <string>Ок</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_button">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../irspy/qt/resources/icons.qrc"/>
 </resources>
 <connections/>
</ui>
